sdk/analysisservices/Azure.ResourceManager.Analysis/api/Azure.ResourceManager.Analysis.netstandard2.0.cs

```diff
@@ -0,0 +1,274 @@
+namespace Azure.ResourceManager.Analysis
+{
+    public static partial class AnalysisExtensions
+    {
+        public static Azure.Response<Azure.ResourceManager.Analysis.Models.AnalysisServerNameAvailabilityResult> CheckNameAvailabilityServer(this Azure.ResourceManager.Resources.SubscriptionResource subscriptionResource, Azure.Core.AzureLocation location, Azure.ResourceManager.Analysis.Models.AnalysisServerNameAvailabilityContent content, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Please rename this method to `CheckAnalysisServerNameAvailability`

Reply:
fixed